<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>789</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>889</width>
    <height>789</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DNF伤害模拟器V1.0  -  by skander820</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget_roles">
      <property name="toolTipDuration">
       <number>-10</number>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabWidget_role0">
       <attribute name="title">
        <string/>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabWidget_add">
       <attribute name="title">
        <string>+</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="files">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="separator"/>
    <addaction name="actionquit"/>
   </widget>
   <widget class="QMenu" name="tools">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionhelp"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionhelp_2"/>
   </widget>
   <addaction name="files"/>
   <addaction name="tools"/>
   <addaction name="help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionquit">
   <property name="text">
    <string>quit</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="text">
    <string>help</string>
   </property>
  </action>
  <action name="actionhelp_2">
   <property name="text">
    <string>help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
